plugin-server/src/main/ingestion-queues/batch-processing/each-batch-onevent.ts

```diff
@@ -59,12 +47,8 @@ export async function eachMessageAppsOnEventHandlers(
 ): Promise<void> {
     const pluginConfigs = queue.pluginsServer.pluginConfigsPerTeam.get(clickHouseEvent.team_id)
     if (pluginConfigs) {
-        // Split between onEvent and composeWebhook plugins
-        const onEventPlugins = pluginConfigs.filter((pluginConfig) => pluginConfig.method === PluginMethod.onEvent)
```

Comment:
Did this (or an equivalent) get moved somewhere else? I'm having trouble finding it, and it's unclear to me whether this was importance (even if only for wasting a lot of cycles, or something)?

Reply:
Yeah this code is so heavily nested that its tricky to follow 😅 
Essentially by removing some of the other code paths, we didn't need to know about the onEventPlugins here and if you go all the way down the stack you eventually get here https://github.com/PostHog/posthog/pull/21983/files/4682aeb8fec6cb1376a5ca8b06edd38c67aa69bb#diff-1b8472934c778571069622f041eaeac4bd00fdd66f38c98048570497b8836a8aR71 which is doing the filtering as well making this line redundant